devdocs/settingUp.md

```diff
@@ -51,7 +51,7 @@ Important: When a version is specified, please install that version instead of t
    If you want to use Sendgrid for developing and testing email features, create a free SendGrid account and update your username and password in `build.properties`
    * `src/test/resources/test.properties`<br>
    Create it using `test.template.properties`. 
-   For now, property values can remain as they are.<br>
+   Append a unique id (e.g your name) to each of the default accounts found at the bottom of this file. e.g change alice.tmms to alice.tmms.&lt;yourName&gt;<br>
```

Comment:
you mean instead of <yourName>? or give an example of an entire line in the file that has to be modified?